### PR TITLE
Fix bootnodes toggle

### LINKED ISSUE #8902: Can't add custom bootnode, toggle is broken
# Problem

Can't add custom bootnode, toggle is broken

## Steps

1. open status
2. tap profile
3. tap advanced
4. tap bootnodes
5. tap toggle on/off -> toggle is NOT enabled
6. add a custom bootnode and save ( for example enode://a8a97f126f5e3a340cb4db28a1187c325290ec08b2c9a6b1f19845ac86c46f9fac2ba13328822590fd3de3acb09cc38b5a05272e583a2365ad1fa67f66c55b34@167.99.210.203:30404)

Actual result: bootnode is not saved, toggle could not be enabled

image

## Notes:

build: nightly 03-09-2019 https://status-im.ams3.digitaloceanspaces.com/StatusIm-190903-025900-df3473-nightly.apk
device: ios 12.4, android 8

### PR DESCRIPTION
**Pull request by cammellos**
Fixes: #8902 

status: wip

### CODE REVIEW

File: src/status_im/subs.cljs [hunk 1]
```diff
@@ -798,14 +798,14 @@
  :settings/bootnodes-enabled
  :<- [:multiaccount]
  :<- [:networks/current-network]
- (fn [{:keys [settings]} current-network]
+ (fn [[{:keys [settings]} current-network]]
```
Comment (flexsurfer) [inline src/status_im/subs.cljs:803]: i hate this mistake! such a pita

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/subs.cljs (hunk 1 above)
Comment (yenda) [reply src/status_im/subs.cljs:803]: could be solved with a macro

**Review by yenda — APPROVED** [review]

**Comment by yenda** [comment]
@cammellos did you check that it was just the toggle? I am not sure the setting itself will actually work

**Comment by cammellos** [comment]
@yenda I have checked that bootnodes are saved and that the app connects if custom bootnodes are enabled,  I can check if they are actually passed in the config if necessary

**Comment by yenda** [comment]
@cammellos I'm pretty sure it is not the case because I don't see any call for `saveNodeConfig`, it should do the same thing as `save-network-settings` calling `node/prepare-new-config`

**Comment by cammellos** [comment]
@yenda thanks,  I will have a look then

**Pull request merged**